retrofit/src/test/java/retrofit/CallTest.java

```diff
@@ -245,8 +245,11 @@
     });
     assertTrue(latch.await(2, SECONDS));
 
-    assertThat(failureRef.get()).isInstanceOf(UnsupportedOperationException.class)
-        .hasMessage("I am broken!");
+    Throwable failure = failureRef.get();
+
+    assertThat(failure).hasMessage("Can not create call for Service.postString(String), HTTP method = POST, relative path template = /");
+    assertThat(failure).hasCauseExactlyInstanceOf(UnsupportedOperationException.class);
```

Comment:
You're already asserting on `getCause()` below, just put a normal `.isInstanceOf` in that chain.

Also, this is a damn ridiculously specific method for AssertJ to have...